Makefile

```diff
@@ -17,7 +21,7 @@ lint:
 	exit
 
 .PHONY: build
-build:
+build: fmt
```

Comment:
You don't need to add `fmt` here and on `test`
It just makes the process of running the test and build slower and it is not necessary

Reply:
removing from test but still feel it is good to use in build as it won't take more than a few additional seconds. This will be useful for avoiding unnecessary additional comments on PR review.